### ISSUE TITLE
Patapon 2 EU - Too fast - Menu

### ISSUE THREAD

**Issue by Astuceman** [hrydgard/ppsspp#9842]
I found a small bug
I don't think it's real speed
It only affect experience 

My settings are by default 
Only show FPS and Fullscreen activated

http://gph.is/2tQ6YxL
![8vixro7pfclhq](https://user-images.githubusercontent.com/19632209/28210548-3886d2f2-6898-11e7-8d00-b178633ec5a4.gif)

**Comment by lasers** [comment]
![patapon2](https://user-images.githubusercontent.com/852504/28241901-d2ec0ddc-6963-11e7-82df-8d60653c59ee.gif)
Source: https://www.youtube.com/watch?v=ganuZciSuPI

**Comment by hrydgard** [comment]
Right, I noticed this quite a while ago, but maybe never created an issue. Definitely a bug, but indeed it seems to only affect the title screen.

**Comment by LunaMoo** [comment]
That screen runs at double framerate, adding it to [DrawSyncEatCycles] hack fixes it, so it's just another timing issue.

**Comment by Michal1324** [comment]
I was aware of this as well but I didn't bother to create it too

**Comment by benderscruffy** [comment]
should this be closed now ??

**Comment by hrydgard** [comment]
Probably not, if it still happens.

**Comment by LatusPL** [comment]
I found a solution to fix this and also maybe recreate it, So if fps in Patapon 2 is highier than 30, like 60 fps here, then go to settings, system, and find PSP CPU Clock... If its Auto, then it selects the best CPU Clock for the game, so go back and make game configuration instead and go backt here and change the PSP CPU Clock to ex. 50 [ I have 56 ], cuz that worked for Me
![image](https://github.com/user-attachments/assets/50f567d2-060d-4513-b524-e6efa86cf25e)
